### PR TITLE
bump to ~4.11.0 in JS/TS samples & yo botbuilder, cleanup TS sample 58

### PR DESCRIPTION
**Pull request by stevengum**

### CODE REVIEW

File: samples/javascript_nodejs/16.proactive-messages/package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
         "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
-        "botbuilder-dialogs": "~4.10.3",
+        "botbuilder": "~4.11.0",
+        "botbuilder-dialogs": "~4.11.0",
         "dotenv": "^8.2.0",
         "fetch-ponyfill": "^6.1.0",
```
Comment (sgellock) [inline samples/javascript_nodejs/16.proactive-messages/package.json:22]: fetch-ponyfill is currently at 7.0.0

File: samples/javascript_nodejs/16.proactive-messages/package.json [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
         "moment": "^2.26.0",
```
Comment (sgellock) [inline samples/javascript_nodejs/16.proactive-messages/package.json:23]: Moment is currently at 2.29.1

**Review by johnataylor — APPROVED** [review]

File: samples/typescript_nodejs/02.echo-bot/package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
+        "botbuilder": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/02.echo-bot/package.json:23]: replace is at 1.2.0

File: samples/typescript_nodejs/03.welcome-users/package.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
+        "botbuilder": "~4.11.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/03.welcome-users/package.json:22]: replace is at 1.2.0

File: samples/typescript_nodejs/05.multi-turn-prompt/package.json [hunk 5]
```diff
@@ -18,8 +18,8 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
-        "botbuilder-dialogs": "~4.10.3",
+        "botbuilder": "~4.11.0",
+        "botbuilder-dialogs": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/05.multi-turn-prompt/package.json:24]: replace is outdated

File: samples/typescript_nodejs/06.using-cards/package.json [hunk 6]
```diff
@@ -18,8 +18,8 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
-        "botbuilder-dialogs": "~4.10.3",
+        "botbuilder": "~4.11.0",
+        "botbuilder-dialogs": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/06.using-cards/package.json:24]: update replace

File: samples/typescript_nodejs/16.proactive-messages/package.json [hunk 7]
```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
+        "botbuilder": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/16.proactive-messages/package.json:23]: update replace

File: samples/typescript_nodejs/50.teams-messaging-extensions-search/package.json [hunk 8]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "botbuilder": "~4.10.3",
+    "botbuilder": "~4.11.0",
     "dotenv": "^8.2.0",
     "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/50.teams-messaging-extensions-search/package.json:24]: update replace

File: samples/typescript_nodejs/50.teams-messaging-extensions-search/package.json [hunk 9]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
```
Comment (sgellock) [inline samples/typescript_nodejs/50.teams-messaging-extensions-search/package.json:21]: axis is at 0.21.0

File: samples/typescript_nodejs/51.teams-messaging-extensions-action/package.json [hunk 10]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
```
Comment (sgellock) [inline samples/typescript_nodejs/51.teams-messaging-extensions-action/package.json:21]: axis is at 0.21.0

File: samples/typescript_nodejs/51.teams-messaging-extensions-action/package.json [hunk 11]
```diff
@@ -19,7 +19,7 @@
...
-    "botbuilder": "~4.10.3",
+    "botbuilder": "~4.11.0",
     "dotenv": "^8.2.0",
     "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/51.teams-messaging-extensions-action/package.json:24]: replace is outdated

File: samples/typescript_nodejs/57.teams-conversation-bot/package.json [hunk 12]
```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
+        "botbuilder": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/57.teams-conversation-bot/package.json:23]: update replace

File: samples/typescript_nodejs/58.teams-start-new-thread-in-channel/package.json [hunk 13]
```diff
@@ -18,8 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.10.3",
-        "botframework-schema": "^4.10.3",
+        "botbuilder": "~4.11.0",
         "dotenv": "^8.2.0",
         "replace": "~1.1.1",
```
Comment (sgellock) [inline samples/typescript_nodejs/58.teams-start-new-thread-in-channel/package.json:23]: update replace

**Review by sgellock — COMMENTED** [review]
see comments

**Pull request merged**